.github/workflows/build-complete-samples.yml

```diff
@@ -871,8 +863,8 @@ jobs:
               name: 'bot-formatting-cards'
               version: '18.x'
 
-            - project_path: 'samples/meeting-tabs/nodejs'
-              name: 'meeting-tabs'
+            - project_path: 'samples/meetings-tabs/nodejs'
+              name: 'meetings-tabs'
               version: '18.x'
 
```

Comment:
You have removed "Connector-github-notification" and other samples, so make sure that all the references from Samples root readme, and Microsoft Documentation has also removed.

Also, please run build pipeline before merging the PR.